xblock/test/test_runtime.py

```diff
@@ -650,28 +649,6 @@ def test_missing_definition(self):
             self.runtime.get_block(self.usage_id)
 
 
-class TestRuntimeDeprecation(WarningTestMixin, TestCase):
```

Comment:
Why was this whole test class removed? It looks like the FieldData deprecation warnings should still be happening.